packages/tina-graphql/src/datasources/data-manager.ts

```diff
@@ -90,7 +89,11 @@ export class DataManager implements DataSource {
     return documents.map((relativePath) => p.join(section.path, relativePath))
   }
   getAllTemplates = async () => {
-    const fullPath = p.join(this.rootPath, tinaPath, 'front_matter/templates')
+    const fullPath = p.join(
+      this.rootPath,
+      TINA_GENERATED_PATH,
+      'front_matter/templates'
```

Comment:
I don't think this will work on windows. Maybe 
```
    const fullPath = p.join(
      this.rootPath,
      TINA_GENERATED_PATH,
      'front_matter',
      'templates'
//...
```